earth2studio/data/utils.py

```diff
@@ -43,8 +44,12 @@ def fetch_data(
     variable: VariableArray,
     lead_time: LeadTimeArray = np.array([np.timedelta64(0, "h")]),
     device: torch.device = "cpu",
+    interp_to: CoordSystem = None,
+    interp_method: str = "nearest",
```

Comment:
Not the biggest fan of the increase of API parameters here. If these are specific for strom cast is this needed outside of call function of stormcast?

Reply:
These should support interpolation for any data source onto any target coordinates; I think this use case will be increasingly common as more flavors of CorrDiff/StormCast emerge and as people play with their own regional datasets. The main usage pattern is intended to have this called at the start of a workflow, before running the model -- fetch data, interp, and move to GPU, then run inference. StormCast just makes use of this in the forward pass to do the regridding of conditioning data on the fly

Reply:
I see. Sounds reasonable, I'm not losing sleep over this so all good. Its okay if these util methods evolve over time to match more and more with whats needed. :+1: